src/Components/Profile/MyPosts/myPosts.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import Post from './Post/post'
+
+import s from './myPosts.module.css';
+
+const MyPosts = () => {
+  return (
+    <div>
+      My posts
+        <div></div>
+      <textarea></textarea>
```

Comment:
I think there's a problem with indentation - don't forget to press "Shift + Alt + F".